From: =?utf-8?q?Rapha=C3=ABl_Hertzog?= <raphael@offensive-security.com>
Date: Mon, 19 Aug 2019 09:12:19 +0200
Subject: Don't include private json-c header

It disappeared with json-c 0.13 and the package fails to build.
---
 manipulators/mwebymsg/analyse.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/manipulators/mwebymsg/analyse.c b/manipulators/mwebymsg/analyse.c
index 1d4c333..8ad7c0e 100644
--- a/manipulators/mwebymsg/analyse.c
+++ b/manipulators/mwebymsg/analyse.c
@@ -32,7 +32,6 @@
 #include <signal.h>
 #include <unistd.h>
 #include <json-c/json.h>
-#include <json-c/json_object_private.h>
 
 #include "log.h"
 #include "analyse.h"
